wolfcrypt/test/test.c

```diff
@@ -27625,6 +27625,20 @@ WOLFSSL_TEST_SUBROUTINE wc_test_ret_t hpke_test(void)
     #endif
 #endif
 
+/* added this code to hit code coverage on lines 219-225 on hpke.c*/
```

Comment:
Nice code comment. Line numbers change, so it would be better to say something like:

```
/* Test coverage for wc_HpkeInit with Curve448, even though it is not supported */
```